.github/workflows/scripts/run-scraper.py

```diff
@@ -0,0 +1,124 @@
+import sys
+import json
+import logging
+import argparse
+import traceback
+from datetime import (datetime, timedelta)
+from functools import partial
+from typing import (List, Optional)
+from cdp_scrapers import instances
+
+
+###############################################################################
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(levelname)4s: %(module)s:%(lineno)4s %(asctime)s] %(message)s",
+)
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+RESULTS_FILE = "run-scraper-results.md"
+
+###############################################################################
+
+
+def iso_strptime(dt: str) -> Optional[datetime]:
+    try:
+        return datetime.strptime(dt, "%Y-%m-%dT%H:%M:%S")
+    except (TypeError, ValueError):
+        try:
+            # try with fractional second
+            return datetime.strptime(dt, "%Y-%m-%dT%H:%M:%S.%f")
+        except (TypeError, ValueError):
```

Comment:
I would just try with `datetime.fromisoformat` [stdlib docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat).

Handles more cases. If it fails then by all means return `None`

Reply:
I thought for sure there has to be the inverse of `isoformat()`. Looked on the `datetime` several times. Alas, `fromisoformat()` was introduced in 3.7. I have 3.6 documentation bookmarked because that's one I use at work. 😅